Cargo.toml

```diff
@@ -41,11 +42,6 @@ validate = []
 name = "criterion"
 harness = false
 
-# MSRV warnings:
-# - arbitrary 1.0.3 requires Rust-1.51
-# - bitflags 1.3 requires Rust-1.46
-# - indexmap 1.7 requires Rust-1.49
```

Comment:
Can we bump `indexmap` to 1.8?

Reply:
It's fine if they have a lower MSRV than us.